go/cmd/gh-ost/main.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/github/gh-ost/go/base"
 	"github.com/github/gh-ost/go/logic"
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
Should we add a comment explaining why we're importing this?

Reply:
I don't believe so. Not use to seeing comments within import statements, and this brings the usage much closer to the standard usage described in the [go-sql-driver/mysql](https://github.com/go-sql-driver/mysql#usage) documentation. Potentially we could move this into the one of the classes where `database/sql` is being imported to exactly match the documentation, but that seems less discoverable to me than adding here in main.